spec/operators/switchLatest-spec.js

```diff
@@ -3,15 +3,166 @@ var Rx = require('../../dist/cjs/Rx');
 var Observable = Rx.Observable;
 var immediateScheduler = Rx.Scheduler.immediate;
 
-describe('Observable.prototype.switchLatest()', function () {
+fdescribe('Observable.prototype.switchLatest()', function () {
```

Comment:
Oops. Good catch.

That's to limit jasmine tests to just that grouping during development. I meant to remove that.